scalanet/src/io/iohk/scalanet/crypto/package.scala

```diff
@@ -0,0 +1,58 @@
+package io.iohk.scalanet
+
+import java.math.BigInteger
+import java.security.SecureRandom
+
+import org.spongycastle.asn1.sec.SECNamedCurves
+import org.spongycastle.asn1.x9.X9ECParameters
+import org.spongycastle.crypto.AsymmetricCipherKeyPair
+import org.spongycastle.crypto.digests.SHA256Digest
+import org.spongycastle.crypto.generators.ECKeyPairGenerator
+import org.spongycastle.crypto.params.{ECDomainParameters, ECKeyGenerationParameters, ECPrivateKeyParameters, ECPublicKeyParameters}
+import org.spongycastle.crypto.signers.{ECDSASigner, HMacDSAKCalculator}
+
+package  object crypto {
```

Comment:
did you copy this code from mantis right? If it that the case, i suggest you to add a comment regarding that, try to add a link to the source file where the code is.